lib/const.rb

```diff
@@ -115,8 +115,14 @@
   CREW_BUILD = 'x86_64-cros-linux-gnu'
 end
 
-CREW_COMMON_FLAGS = '-O2 -pipe -flto -ffat-lto-objects -fPIC -fuse-ld=gold'
-CREW_COMMON_FNO_LTO_FLAGS = '-O2 -pipe -fno-lto -fPIC -fuse-ld=gold'
+case ARCH
+when 'aarch64', 'armv7l'
+  CREW_LINKER = 'gold'
+when 'i686', 'x86_64'
+  CREW_LINKER = 'mold'
+end
```

Comment:
Is this something we could override with an environment variable?

Reply:
I intentionally labeled it `CREW_LINKER` with that in mind...

Reply:
But you can always do a `no_env_options` and then do a `CREW_ENV_OPTIONS.tr('mold','lld')` or whatever...

Reply:
Too hackish. :)

Reply:
is the new logic better?